src/components/ItemTable.tsx

```diff
@@ -0,0 +1,91 @@
+import "./TagStyle.css";
+import { DataGrid, GridColDef, GridRowsProp } from "@mui/x-data-grid";
+import { DataStore } from "@aws-amplify/datastore";
+import { useEffect } from "react";
+// import TagsList from "./TagsList";
+import { Item } from "../models";
+
+// Sample data, should remove it after the fetch call is fixed
+// REMARKS: the type of categorys here doesn't matched with the data
+const rows: GridRowsProp = [
+  {
+    id: 1,
+    name: "Air Jordan",
+    dataAdded: "2/14/2023",
+    value: 60,
+    image: "",
+    categorys: ["Clothing", "tag3", "tag4"],
+    createdAt: "2023-03-12T01:32:37.785Z",
+    updatedAt: "2023-03-12T01:32:37.785Z",
+  },
+  {
+    id: 2,
+    name: "New balance sneaker",
+    dataAdded: "2/23/2023",
+    value: "$60",
+    image: "",
+    categorys: ["Alothing", "tag2", "tag3", "tag4", "tag5"],
+    createdAt: "2023-03-13T01:32:37.785Z",
+    updatedAt: "2023-03-13T01:32:37.785Z",
+  },
+];
+
+const columns: GridColDef[] = [
+  { field: "name", headerName: "Name", minWidth: 200 },
+  { field: "value", headerName: "Price", minWidth: 150 },
+  { field: "dataAdded", headerName: "Date Added", minWidth: 150 },
+  { field: "createdAt", headerName: "Created At", minWidth: 200 },
+  { field: "updatedAt", headerName: "Updated At", minWidth: 200 },
+  // TODO: put the Categorys back once the fetch call is fixed
```

Comment:
We will have to update this relationship to be in proper english. Thanks for finding this